backend/consensus/base.py

```diff
@@ -335,18 +339,62 @@ async def exec_transaction(
             )
         )
 
+    def commit_reveal_accept_transaction(
+        self,
+        transaction: Transaction,
+        transactions_processor: TransactionsProcessor,
+    ):
+        # temporary, reuse existing code
+        # and add other possible states the transaction can go to
+        ConsensusAlgorithm.dispatch_transaction_status_update(
+            transactions_processor,
+            transaction.hash,
+            TransactionStatus.COMMITTING,
+            self.msg_handler,
+        )
+        transactions_processor.create_rollup_transaction(transaction.hash)
+
+        ConsensusAlgorithm.dispatch_transaction_status_update(
+            transactions_processor,
+            transaction.hash,
+            TransactionStatus.REVEALING,
+            self.msg_handler,
+        )
+        transactions_processor.create_rollup_transaction(transaction.hash)
+
+        time.sleep(2)  # remove this
+
+        transactions_processor.set_transaction_timestamp_accepted(transaction.hash)
```

Comment:
@kstroobants can we do this set_transaction_timestamp_accepted inside the dispatch_transaction_status_update?

Reply:
If it is called dispatch_transaction_update then maybe yes. I could add a flag in the arguments to update the timestamp or not update it.

But actually, I want to keep them separate because I don't update this timestamp a lot compared to the amount of times I call dispatch. It makes it more complicated for the reader of the code if I would combine them.